{"name": "Unsuccessful login", "status": "passed", "steps": [{"name": "Error message is visible", "status": "passed", "start": 1737700077096, "stop": 1737700077096}], "parameters": [{"name": "username", "value": "'standard_user'"}, {"name": "password", "value": "''"}, {"name": "driver", "value": "'Edge'"}], "start": 1737700076865, "stop": 1737700077131, "uuid": "3bb8527f-f602-4918-8139-81ea975579b5", "historyId": "835dedd962576e463d3e402931bc7f9f", "testCaseId": "b876685be5aab7257716ff999cc789c0", "fullName": "Tests.test_Login.TestLogin#test_1", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az1704-590"}, {"name": "thread", "value": "1930-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}